content/_index.html

```diff
@@ -0,0 +1,407 @@
+---
+title: RAPIDS.ai | GPU accelerated data science 
+description: homepage
+---
+
+  <!-- Hero Shortcode -->
+  {{< hero hero-text="GPU Accelerated Data Science" button-link="#quick-start" button-text="Quick Start" >}}
+
+  <!-- Announcements Shortcode (0-3x)-->
+  <!-- UPDATE HERE -->
+  {{< announcements text-1="22.12 Released!" link-1="https://docs.rapids.ai/notices" text-2="CUDA 11.8 Support 11.12.22" link-2="https://docs.rapids.ai/notices" text-3="Ubuntu 18.04 support dropped" link-3="https://docs.rapids.ai/notices" >}}
+
+  <!-- Intro -->
+  <section class="section padding-bottom-slant">
+    <div class="container">
+
+      <h1 class="has-text-centered mb-5">Data Science at the speed of thought</h1>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-caret-right"></i> What is RAPIDS </h2>
+          <p> RAPIDS is unmatched speed with closely-matched APIs of over a dozen libraries. Built on the shoulders of giants including <a href="https://developer.nvidia.com/cuda-toolkit" target="_blank">NVIDIA CUDA</a> and <a href="https://arrow.apache.org/" target="_blank">Apache Arrow</a>, unlock the speed of GPUs with code you already know. </p>
+           <a href="/learn-more/#about">Learn more on the About Section <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-bolt-lightning"></i> Why Use RAPIDS</h2>
+          <p> RAPIDS allows fluid, creative interaction with data for everyone from BI users to AI researchers on the cutting edge. Unmatched speed means less time and less cost moving data and training models. </p>
+          <a href="/learn-more/#use-cases">Find out more from RAPIDS Use Cases <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-light fa-chart-network"></i> Open Source Ecosystem </h2>
+          <p>RAPIDS is Open Source and available on <a href="https://github.com/rapidsai" taget="_blank">GitHub</a>. Our mission is to empower and advance the open-source GPU data science ecosystem and work toward flawless integration. </p>
+          <a href="https://github.com/rapidsai">Jump to RAPIDS on GitHub <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+      </div>
+
+      <div class="columns mt-5">
+        <div class="column is-one-third">
+          <h2><i class="fa-thin fa-objects-column"></i> Faster Pandas <br> with cuDF</h2>
+          <p>cuDF is nearly a drop in replacement to Pandas for most use cases and has greatly improved performance.</p>
+
+          <canvas id="cuDFchart" width="400" height="200"></canvas>
+          <p>*A100 vs i7-6000</p>
```

Comment:
Graphs look great, but axes are unclear. Maybe add "(times in seconds, lower is faster)"

Definitely want to add the name of the operation too like "

"A100 vs. i7, 100M row join benchmark, times in seconds"